oak_containers_orchestrator/src/ipc_server.rs

```diff
@@ -54,19 +51,6 @@ impl Orchestrator for ServiceImplementation {
         Ok(tonic::Response::new(()))
     }
 
-    async fn get_attestation_evidence(
```

Comment:
Does it mean that the enclave app won't be able to request it's own evidence?

Reply:
Yes, I don't think we have any plans that require this on Oak Containers.